docs/reference/search/rrf.asciidoc

```diff
@@ -32,10 +32,11 @@ return score
 ==== Reciprocal rank fusion API
 
 You can use RRF as part of a <<search-search, search>> to combine and rank
-documents using multiple result sets from
-
-* 1 query and 1 or more kNN searches
-* 2 or more kNN searches
+documents using result sets from a combination of
+<<request-body-search-query, query>>,
+<<request-body-sub-searches, sub searches>>, and/or
+<<search-api-knn, knn searches>>. A minimum of 2 results sets for
```

Comment:
should we drop the last word "for" here?

Reply:
you are correct .`for` should be removed

Reply:
Good catch! Fixed.